dependencies/prepare.sh

```
@@ -1,32 +1,27 @@
-  libtool \
-  m4 \
-  pkg-config \
-  cmake
+apt-get install -y curl crossbuild-essential-${DEBIAN_ARCH} libc-dev:${DEBIAN_ARCH} autoconf automake libtool m4 pkg-config cmake
 
-dpkg-architecture -a "$DEBIAN_ARCH" >/env
-echo "PKG_CONFIG_PATH=/usr/lib/${DEB_HOST_MULTIARCH}/pkgconfig" >>/env
-# shellcheck disable=SC2046
+dpkg-architecture -a $DEBIAN_ARCH >/env
+echo PKG_CONFIG_PATH=/usr/lib/$(dpkg-architecture -q DEB_HOST_MULTIARCH)/pkgconfig >>/env
 export $(cat /env)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)